libs/ui/src/widgets/progress.ts

```
@@ -0,0 +1,500 @@
+        </div>
+      </div>`;
+
+    case 'text':
+    default:
+      const linesHtml = Array(lines)
+        .fill(0)
+        .map((_, i) => {
+          const lineWidth = i === lines - 1 ? '60%' : i === 0 ? '90%' : '80%';
+          return `<div class="bg-gray-200 h-4 rounded" style="width: ${lineWidth}"></div>`;
+        })
+        .join('\n');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)